check-compatibility-with-govuk-frontend-version.js

```diff
@@ -0,0 +1,49 @@
+if (process.env.HMRC_FRONTEND_DISABLE_COMPATIBILITY_CHECK === 'true') {
+  process.exit(0);
+}
+const fs = require('fs');
+const path = require('path');
+
+// To our knowledge hmrc-frontend won't work with versions below this.
+const minimumGovukFrontendVersion = '5.4.0';
+
+function semverGreaterThan(a, b) {
+  const [aMaj, aMin, aPat] = a.split('.').map(Number);
+  const [bMaj, bMin, bPat] = b.split('.').map(Number);
+  return ((aMaj - bMaj) || (aMin - bMin) || (aPat - bPat)) > 0;
+}
+
+const projectDir = process.env.INIT_CWD;
+const projectPackagePath = path.join(projectDir, 'package.json');
```

Comment:
I think for these we have the risk of an exception being thrown too, because INIT_CWD can be undefined

wonder if we should wrap the whole file in a try catch and exit(0) on any exception so it's safe by default